src/Uno.Extensions.Navigation.Toolkit/Uno.Extensions.Navigation.Toolkit.UI.csproj

```
@@ -19,7 +19,22 @@
 		<PackageReference Include="Uno.UI" Version="4.3.6" />
 		<PackageReference Include="Uno.Toolkit.UI" Version="2.0.0" />
 	</ItemGroup>
-	
+
 	<Import Project="common.props" />
 
+	<!--
+	Fix for: https://github.com/unoplatform/uno.extensions/issues/563 
+	MSBuild Extras excludes .xf.xml layout files from referenced libraries at
+	https://github.com/novotnyllc/MSBuildSdkExtras/blob/252ba9af3300db98fde54644316567f0c6426e1e/Source/MSBuild.Sdk.Extras/Build/Workarounds.targets#L94
+	-->
+	<Target Name="FixXrXml"
+						AfterTargets="_SdkIncludeWindowsLibraryLayoutItems"
+						Condition="'$(GenerateLibraryLayout)' == 'true' AND '$(SDKIdentifier)' == 'Windows'">
+		<Message Importance="high" Text="Fix for missing .xr.xml files" />
```

```suggestion
		<Message Importance="high" Text="Applying fix for missing .xr.xml files" />
```
or, remove the message?